plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -143,6 +144,15 @@ func (s *serviceAccount) Admit(a admission.Attributes) (err error) {
 		return nil
 	}
 
+	updateInitialized, err := util.IsUpdatingInitializedObject(a)
+	if err != nil {
+		return err
+	}
+	if updateInitialized {
```

Comment:
`IsUpdatingInitializedObject` returns true only if `oldObj.initializers==nil`, i.e., false if the pod is already initialized before the update.

I'll correct the comment of the function.